<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-RRJ270U" failures="1" tests="6" name="Login Test on browser" time="21.608" errors="0" timestamp="2023-06-17T20:29:34 ICT">
  <testcase classname="quangtester.testcase.LoginTest" name="loginTestDataProvider" time="4.482">
    <failure type="java.lang.AssertionError" message="Login không thành công expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login không thành công expected [true] but found [false]
at quangtester.pages.LoginPage.login(LoginPage.java:53)
at quangtester.testcase.LoginTest.loginTestDataProvider(LoginTest.java:24)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- loginTestDataProvider -->
  <testcase classname="quangtester.testcase.LoginTest" name="loginTestFailure" time="8.165"/>
  <testcase classname="quangtester.testcase.LoginTest" name="loginTestExcel" time="10.243"/>
  <testcase classname="quangtester.testcase.LoginTest" name="loginTestDataProviderExcel" time="10.726"/>
  <testcase classname="quangtester.testcase.LoginTest" name="loginTestSuccessExcel" time="10.956"/>
  <testcase classname="quangtester.testcase.LoginTest" name="loginTestDataProvider" time="9.333"/>
</testsuite> <!-- Login Test on browser -->
